src/machine/machine_rp2_usb.go

```diff
@@ -149,6 +162,37 @@ func sendStallViaEPIn(ep uint32) {
 	_usbDPSRAM.EPxBufferControl[ep&0x7F].In.Set(val)
 }
 
+// Set ENDPOINT_HALT/stall status on a USB OUT endpoint.
+func (dev *USBDevice) SetStallEPOut(ep uint32) {
+	if ep == 0 {
```

Comment:
Should `ep` be masked before comparing with 0?

Reply:
The 0 endpoint is a special case. It's both an in and out endpoint in one so there's no direction bit applied for it, but couldn't hurt